packages/ui/tsup.config.ts

```diff
@@ -52,5 +53,6 @@ export default defineConfig(overrideOptions => {
     minify: false,
     sourcemap: true,
     esbuildPlugins: [tailwindcssTransformerCode],
+    treeshake: true,
```

Comment:
This enables rollup's tree shaking instead of just relying on the limited tree shaking built into esbuild.